### ISSUE TITLE
Error importing the postalCodeAutocompleteModule

### ISSUE THREAD

**Issue by Oladian** [albertodam/postal-code-autocomplete#2]
`ERROR in Unexpected value 'PostalCodeAutocompleteModule in /application/node_modules/postal-code-autocomplete/lib/postal-code-autocomplete.module.d.ts' imported by the module 'MyModule in /application/src/app/pages/...'. Please add a @NgModule annotation.`

**I just added the module in the imports section of my module and it seems doesn't work.**

Example:
```
import { PostalCodeAutocompleteModule } from 'postal-code-autocomplete';

@NgModule({
  declarations: [
    SomeComponents,
    ...
  ],
  imports: [
    OtherModules
    PostalCodeAutocompleteModule,
  ],
  exports: [
    SomeComponents,
    ...
  ],
  providers: [
     SomeServicesAndModules...
  ]
})
export class MyModule { ... }
```

Another error thrown in the template where I import the component:

```
'ng-postal-code-autocomplete' is not a known element:
1. If 'ng-postal-code-autocomplete' is an Angular component, then verify that it is part of this module.
2. If 'ng-postal-code-autocomplete' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.ng(0)
```

**Comment by albertot-dev** [comment]
ok i forgot to add custom schema to my component. The library is already updated, I hope it works.

**Comment by albertot-dev** [comment]
resolved version 0.0.6
